app/controllers/coupons_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class CouponsController < ApplicationController
+
+  def all
+    @coupons = Coupon.all
```

Comment:
This could be a one liner `@coupons = Coupons.all.sort_by(&:id)`

Are they not sorted by I'd anyway though? You could do `.order(id: :desc)` or asc